users_index.ml

```diff
@@ -143,29 +143,42 @@ let users_index_layout (users : User_model.user list) current_time =
                                            ]
                                          [
                                            p [ txt user.name ];
-                                           (match user.active with
-                                           | true ->
-                                               i
-                                                 ~a:
-                                                   [
-                                                     a_class
-                                                       [
-                                                         "text-primary-500 \
-                                                          fa-solid fa-check";
-                                                       ];
-                                                   ]
-                                                 []
-                                           | false ->
-                                               i
-                                                 ~a:
-                                                   [
-                                                     a_class
-                                                       [
-                                                         "text-secondary-500 \
-                                                          fa-solid fa-x";
-                                                       ];
-                                                   ]
-                                                 []);
+                                           (if user.super_user then
```

Comment:
shouldn't we distinguish, also in the super-user case, between being active and not active?

Reply:
Yes I think we can do that.